microsoft-edge/progressive-web-apps-chromium/index.md

```diff
@@ -113,123 +129,60 @@ keywords: progressive web apps, PWA, Edge, JavaScript, Windows, UWP, Microsoft S
-   :::column-end:::
-:::row-end:::
-:::row:::
-   :::column span="1":::
-      [Responsive design][WikiResponsiveWebDesign]
-   :::column-end:::
-   :::column span="2":::
-      Employs fluid layouts and flexible images.  Responsive design includes the following elements that adapt your UX to your user's device.
-
-      *   CSS [grid][MDNCssGridLayout]
-      *   [flexbox][MDNCssFlexibleBoxLayout]
-      *   CSS [grid][MDNCssGridLayout] and [flexbox][MDNCssFlexibleBoxLayout]
-      *   [media queries][MDNMediaQueries]
-      *   [responsive images][MDNResponsiveImages]
-
-      Uses [device emulation tools][DevToolsGuideDeviceModeTestingOtherBrowsers] from your browser to locally test, or create a remote debugging session on [Windows][DevtoolsRemoteDebuggingWindows] or [Android][DevtoolsRemoteDebuggingIndex] to test directly on a target device.
-   :::column-end:::
-:::row-end:::
-:::row:::
-   :::column span="1":::
-      [Deep linking][WikiDeepLinking]
-   :::column-end:::
-   :::column span="2":::
-      Routes each page of your site to a unique URL so existing users may help you engage an even broader audience through social media sharing.
-   :::column-end:::
-:::row-end:::
-:::row:::
-   :::column span="1":::
-      [Validation and testing practices][Webhint]
-   :::column-end:::
-   :::column span="2":::
-      Uses code quality tools like the [Webhint][Webhint] linter to optimize the efficiency, robustness, safety, and accessibility of your app.
-   :::column-end:::
-:::row-end:::
-:::row:::
-   :::column span="1":::
-      [Chromium PWA Checklist][WebDevGoodPwaChecklist]
-   :::column-end:::
-   :::column span="2":::
-      Verifies your PWA against the Google baseline PWA checklist.
-   :::column-end:::
-:::row-end:::
+As the most used app on PCs, the Microsoft Store provides a trustworthy and familiar experience for your users to install your app. Additionally, it provides you with analytics.
+
+## Success stories
+
+Using PWA technologies is a great way to make your app **safe**, **discoverable**, **linkable**, **easy to install** and update, **responsive**, and **network independent**. Many businesses use PWAs and have had success stories.
 
-> [!NOTE]
-> To turn your PWA into a [Microsoft Store][MicrosoftDeveloperStore] app, navigate to [Progressive Web Apps in the Microsoft Store][PwaChromiumMicrosoftStore].
+*   The Starbucks PWA has increased daily active users twice and orders on desktop are nearly the same rate as mobile ([source][StarbucksSuccessStory]).
+*   Trivago saw an increase of 150% for people who add its PWA to the home screen and the increased engagement led to a 97% increase in clickouts to hotel offers ([source][TrivagoSuccessStory]).
+*   Tinder cut load times from 11.91 seconds to 4.68 seconds with their PWA and the app is 90% smaller than the native Android app ([source][TinderSuccessStory]).
+
+To read more success stories, navigate to [PWA Stats][PwaStats].
```

Comment:
"navigate to" feels so formal. Could we say something like "check out" or "head over to" instead?

Reply:
agree, will message you offline